app/views/events/show.html.erb

```diff
@@ -60,9 +60,13 @@
         data: { confirm: 'Are you sure? This will also attempt to delete the Discord message related to this event.' },
         class: "btn btn-danger" %>
     <% end %>
-    <%= link_to @event.google_calendar_url, class: "btn btn-primary", target: "_blank" do %>
-      <i class="fa fa-calendar"></i> Add to Google Calendar
+   
+    <% if @event.end_date.past? %>
```

Comment:
```suggestion
    <% unless @event.end_date.past? %>
```
My mistake, we should be showing the button unless the end date has passed

Reply:
Make sense! Thanks for helping